SDBlockDevice.cpp

```diff
@@ -147,6 +147,11 @@
 #include "mbed_debug.h"
 #include <errno.h>
 
+/* Required version: 5.5.4 and above */
+#if (MBED_VERSION < MBED_ENCODE_VERSION(5,5,4))
```

Comment:
FYI, Sam suggested to move this macro to mbed-os code, I will try to add PR in this week.